sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -307,6 +307,11 @@ functionCall
     | aggregateFunction (orderByClause)? filterClause               #filteredAggregationFunctionCall
     | relevanceFunction                                             #relevanceFunctionCall
     | highlightFunction                                             #highlightFunctionCall
+    | functionWithCachedValue                                       #functionLikeConstantCall
+    ;
+
+functionWithCachedValue
```

Comment:
`constantValuedFunction` or `constantFunction` are more appropriate names for this rule.

Reply:
Thank you, renamed in 6ef778de.